### DISCUSSION TITLE
Welcome to Greenwood Discussions!

### DISCUSSION THREAD

**Discussion by thescientist13** [ProjectEvergreen/greenwood#501] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aholtzman** [comment]
I am Adam, I help out with some front end work, a little design, and guides. I also ask important questions like "why?" and "have you seen this?".

**Reply by thescientist13** [reply]
Hi @aholtzman , thanks for saying Hi and thanks for being such a valuable member of the team!  Looking forward to making some really awesome stuff with you and @hutchgrant this year!  🚀
